### PR TITLE
Pewpew layout: Dynamic Cascading List

### PR DESCRIPTION
**Pull request by pablo014**
<h1>Dynamic Cascading List</h1>
<h2>Summary</h2>
This list will take in data from the map data and display it onto  the list and work as a feed
<h2>Preview</h2>

Screen Shot 2021-07-22 at 2 36 25 PM
Screen Shot 2021-07-22 at 2 36 41 PM

<h2>Warning: may mess up with poor internet connection</h2>

### CODE REVIEW

File: src/pages/NotFound.vue [hunk 1]
```diff
@@ -10,9 +10,19 @@
 
 <script>
 import HomeLayout from '@/layouts/Home';
+
 export default {
   name: 'NotFound',
   components: { HomeLayout },
+  data() {
```
Comment (deepanchal) [inline src/pages/NotFound.vue:17]: I don't understand why you changed `NotFound.vue`. Were you testing things and just forgot to remove this? :thinking:

File: src/pages/unauthenticated/PewPew.vue [hunk 2]
```diff
@@ -265,23 +405,87 @@ import {
...
 } from '@/utils/map';
 import { HomeNavigation } from '@/components/home/SideNav';
-import Table from '@/components/Table';
 import { getQueryString } from '@/utils/urls';
 import BarChart from '@/components/charts/BarChart';
 import { Sprite, Texture, Graphics, utils as pixiUtils } from 'pixi.js';
 import Incident from '@/models/Incident';
-import { orderBy } from 'lodash';
+import OrganizationActivity from '@/components/OrganizationActivity.vue';
+import _, { orderBy } from 'lodash';
```
Comment (deepanchal) [inline src/pages/unauthenticated/PewPew.vue:413]: No need to import `orderBy` like this. Just use `_.orderBy`
```suggestion
import _ from 'lodash';
```

File: src/pages/unauthenticated/PewPew.vue [hunk 3]
```diff
@@ -723,6 +928,112 @@ export default {
...
+            }
+          } else {
+            difference += ' minutes ago';
+          }
+        } else {
+          difference += ' seconds ago';
+        }
+      }
+      return difference;
+    },
```
Comment (deepanchal) [inline src/pages/unauthenticated/PewPew.vue:1036]: This function might not be working as you expected. You don't even have to write this function. Use `vue-moment` package that we have. 
```vue
this.$moment(myDate).fromNow()

// moment(new Date("12/10.2020")).fromNow()
// "7 months ago"

```
See: https://momentjs.com/

**Review by deepanchal — COMMENTED** [review]
PR looks great :100: 
Please take a look at my comments :)

File: src/pages/unauthenticated/PewPew.vue (hunk 3 above)
Comment (pablo014) [reply src/pages/unauthenticated/PewPew.vue:1036]: done

File: src/pages/NotFound.vue (hunk 1 above)
Comment (pablo014) [reply src/pages/NotFound.vue:17]: lol seems like I did forget to take that out😂

File: src/pages/unauthenticated/PewPew.vue (hunk 2 above)
Comment (pablo014) [reply src/pages/unauthenticated/PewPew.vue:413]: done

**Pull request merged**